cookbook/integration_langchain.ipynb

```diff
@@ -378,15 +378,19 @@
     },
```

Comment:
<p>function name = <code>name</code></p><p> </p><p>remove  <code>update_current_trace</code></p><p> </p><p>add single line comments to get_current_langchain_handler and   chain.invoke</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>